.github/workflows/release.yml

```diff
@@ -10,13 +10,13 @@ on:
 jobs:
   release:
     name: Release
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
Pinning latest can create unintended behavior when a new "latest" is released. Not necessarily a deal-breaker for using it, but something to consider.

Reply:
Sorry, just noticed this was a draft 😅

Reply:
No problem! Appreciate the comment. Pinned it down to 20.04. PR is now ready for review.